api/python/provisioner/commands/upgrade/get_swupgrade_info.py

```diff
@@ -121,6 +132,13 @@ def run(self, release: str = None) -> dict:
         """
         local_minion = local_minion_id()
 
+        if iso_path is not None:
+            # if the `iso_path` is set up, we ignore the `release` parameter
+            # NOTE: get `SetSWUpgradeRepo` from list of commands since all
+            #  objects are correctly defined (post- and pre- states)
+            set_swupgrade_repo_obj = commands['set_swupgrade_repo']
+            return set_swupgrade_repo_obj.run(iso_path, dry_run=True)
```

Comment:
It means we return different result format depending on inputs. Could you update to return the same format to include:
- metadata
- packages

It would be useful for other tasks also. Thanks

Reply:
Done